diff --git a/sharethis.module b/sharethis.module
index e1ff5af..201e974 100644
--- a/sharethis.module
+++ b/sharethis.module
@@ -9,9 +9,9 @@
  *
  * Displays help and module information.
  *
- * @param path 
+ * @param path
  *   Which path of the site we're using to display help
- * @param arg 
+ * @param arg
  *   Array that holds the current path as returned from arg() function
  */
 function sharethis_help($path, $arg) {
@@ -38,7 +38,7 @@ function sharethis_help($path, $arg) {
 function sharethis_permission() {
   return array(
     'administer sharethis' => array(
-      'title' => t('Administer ShareThis'), 
+      'title' => t('Administer ShareThis'),
       'description' => t('Change the settings for how ShareThis behaves on the site.'),
     ),
   );
@@ -60,8 +60,8 @@ function sharethis_admin() {
   drupal_add_js($my_path . '/ShareThisForm.js');
   drupal_add_js($my_path . '/stlib_picker.js');
   drupal_add_css($my_path . '/stlib_picker.css');
- 
-  return drupal_get_form('sharethis_form'); 
+
+  return drupal_get_form('sharethis_form');
 }
 
  /**
@@ -73,7 +73,7 @@ function sharethis_form($form, &$form_state) {
   $my_path = drupal_get_path('module', 'sharethis');
   $current_options_array = sharethis_get_options_array();
   global $base_url;
-  
+
   // Create the variables related to widget choice.
   $widget_type = $current_options_array['widget'];
   $widget_markup = "";
@@ -91,7 +91,7 @@ function sharethis_form($form, &$form_state) {
     $service_string_markup .= "\"" . $key . "\",";
   }
   $service_string_markup = substr($service_string_markup, 0, -1);
-  
+
   // Create an array of node types.
   $node_type_array = node_type_get_types();
   $node_type_options = array();
@@ -105,7 +105,7 @@ function sharethis_form($form, &$form_state) {
   $publisher = $current_options_array['publisherID'];
   // Create the variables for teasers.
   $teaser = $current_options_array['viewMode'] == "1" ? TRUE : FALSE;
-  
+
   $form = array();
   $form['options'] = array(
     '#type' => 'fieldset',
@@ -218,10 +218,10 @@ function sharethis_form($form, &$form_state) {
     '#default_value' => variable_get('sharethis_twitter_suffix', ''),
   );
   $form['submit'] = array(
-    '#type' => 'submit', 
+    '#type' => 'submit',
     '#value' => t('Submit')
   );
-  return $form; 
+  return $form;
 }
 
 /**
@@ -229,17 +229,17 @@ function sharethis_form($form, &$form_state) {
  */
 function sharethis_form_submit($form, &$form_state) {
   // Record the widget option.
-  if (isset($form_state['values']['widget_option'])) {
-    variable_set('sharethis_widget_option', $form_state['values']['widget_option']);
+  if (isset($form_state['input']['widget_option'])) {
+    variable_set('sharethis_widget_option', $form_state['input']['widget_option']);
   }
 
   // Record the buttons option.
-  if (isset($form_state['values']['button_option'])) {
-    variable_set('sharethis_button_option', $form_state['values']['button_option']);
+  if (isset($form_state['input']['button_option'])) {
+    variable_set('sharethis_button_option', $form_state['input']['button_option']);
   }
 
   // Record the services option.
-  $services = isset($form_state['values']['service_option']) ? $form_state['values']['service_option'] : '';
+  $services = isset($form_state['input']['service_option']) ? $form_state['input']['service_option'] : '';
   $services = str_replace(";", "", $services);
   $services = str_replace("\\", "", $services);
   $services = str_replace("/", "", $services);
@@ -248,26 +248,26 @@ function sharethis_form_submit($form, &$form_state) {
   variable_set('sharethis_service_option', $services);
 
   // Save the extra services.
-  $extras = isset($form_state['values']['sharethis_option_extras']) ? $form_state['values']['sharethis_option_extras'] : array();
+  $extras = isset($form_state['input']['sharethis_option_extras']) ? $form_state['input']['sharethis_option_extras'] : array();
   variable_set('sharethis_option_extras', $extras);
 
   // Record the NodeType option.
-  if (isset($form_state['values']['node_option'])) {
-    $types = implode(',', $form_state['values']['node_option']);
+  if (isset($form_state['input']['node_option'])) {
+    $types = implode(',', $form_state['input']['node_option']);
     variable_set('sharethis_node_option', $types);
-  }  
-  
+  }
+
   // Record the teaser option.
-  if (isset($form_state['values']['teaser_option'])) {
-    variable_set('sharethis_teaser_option', $form_state['values']['teaser_option']);
-  } 
+  if (isset($form_state['input']['teaser_option'])) {
+    variable_set('sharethis_teaser_option', $form_state['input']['teaser_option']);
+  }
   else {
     variable_set('sharethis_teaser_option', 0);
   }
 
   // Record the publisher ID option.  Since it's a text field, remove anything that resembles code
-  if (isset($form_state['values']['publisherID'])) {
-    $sanitized = $form_state['values']['publisherID'];
+  if (isset($form_state['input']['publisherID'])) {
+    $sanitized = $form_state['input']['publisherID'];
     $sanitized = str_replace(";", "", $sanitized);
     $sanitized = str_replace(":", "", $sanitized);
     $sanitized = str_replace("\\", "", $sanitized);
@@ -278,21 +278,21 @@ function sharethis_form_submit($form, &$form_state) {
   }
 
   // The location of the widget.
-  if (isset($form_state['values']['sharethis_location'])) {
-    variable_set('sharethis_location', $form_state['values']['sharethis_location']);
+  if (isset($form_state['input']['sharethis_location'])) {
+    variable_set('sharethis_location', $form_state['input']['sharethis_location']);
   }
 
   // The weight of the widget.
-  if (isset($form_state['values']['sharethis_weight'])) {
-    variable_set('sharethis_weight', $form_state['values']['sharethis_weight']);
+  if (isset($form_state['input']['sharethis_weight'])) {
+    variable_set('sharethis_weight', $form_state['input']['sharethis_weight']);
   }
 
-  $sharethis_twitter_suffix = isset($form_state['values']['sharethis_twitter_suffix']) ? $form_state['values']['sharethis_twitter_suffix'] : '';
+  $sharethis_twitter_suffix = isset($form_state['input']['sharethis_twitter_suffix']) ? $form_state['input']['sharethis_twitter_suffix'] : '';
   variable_set('sharethis_twitter_suffix', $sharethis_twitter_suffix);
 
   drupal_set_message(t('The configuration options have been saved.'));
 }
- 
+
  /**
  * Implements hook_menu.
  *
@@ -315,9 +315,9 @@ function sharethis_menu() {
  * Inserts ShareThis widget code onto each node view.
  * TODO: Want to add the option somewhere to select nodes.
  *
- * @param node 
+ * @param node
  *   The node that is being acted upon
- * @param view_mode 
+ * @param view_mode
  *   The type of view (teaser, full, etc)
  * @param langcode
  *   Information about the language
@@ -329,7 +329,7 @@ function sharethis_node_view($node, $view_mode, $langcode) {
   }
   // First get all of the options for the sharethis widget from the database:
   $data_options = sharethis_get_options_array();
-  
+
   // This looks to see if the path variable has been posted by some rewrite module.
   // This is not super efficient, O(N), but N is often less than 20.
   $is_path = FALSE;
@@ -356,12 +356,12 @@ function sharethis_node_view($node, $view_mode, $langcode) {
   // Get the full path to insert into the Share Buttons.
   $mPath = $base_url . $path_module;
   $mTitle = $node->title;
-  
+
   // Only display the ShareThis buttons if this node fits all the requirements
   if (strpos($data_options['nodeType'], $node->type) !== FALSE) { // Make sure this is the right type of node.
     if (($data_options['viewMode'] == "1") && ($view_mode == "teaser")) { // If "don't show for teaser" is enabled, and this is a teaser, don't do anything
       // Do nothing.
-    } 
+    }
     else {
       // Check where we want to display the ShareThis widget.
       switch (variable_get('sharethis_location', 'content')) {
